ui/css/design-system/_typography.scss

```diff
@@ -1,6 +1,6 @@
 @use "sass:map";
 
-$font-family: var(--font-family-sans); // --font-family-sans from @metamask/design-tokens stylesheet
+$font-family: var(--font-family-default); // from @metamask/design-tokens stylesheet
```

Comment:
Replacing removed `--font-family-sans` with `--font-family-default` as per the [migration guide](https://github.com/MetaMask/metamask-design-system/blob/main/packages/design-tokens/MIGRATION.md#from-version-600-to-700)